apps/autofix/src/container-server/userContainer.ts

```diff
@@ -30,9 +31,14 @@ export class UserContainer extends Container<Env> {
 
 	async container_initialize(): Promise<void> {
 		// kill container
-		await this.stopContainer()
+		// await this.stopContainer()
 
-		await this.startAndWaitForPorts(OPEN_CONTAINER_PORT)
+		await pRetry(async () => this.startAndWaitForPorts(OPEN_CONTAINER_PORT), {
```

Comment:
drive by change to maybe make this more reliable. We were having issues with this earlier